sdk/cosmosdb/cosmos/src/request/request.ts

```diff
@@ -1,13 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-import { logger } from "@azure/identity";
 import { setAuthorizationHeader } from "../auth";
 import { Constants, HTTPMethod, jsonStringifyAndEscapeNonASCII, ResourceType } from "../common";
 import { CosmosClientOptions } from "../CosmosClientOptions";
 import { PartitionKey } from "../documents";
 import { CosmosHeaders } from "../queryExecutionContext";
 import { FeedOptions, RequestOptions } from "./index";
-
+import { defaultLogger } from "../common/logger";
```

Comment:
defaultLogger uses "cosmosdb", in some other places using "RequestHandler", "ClientContext", "parallelQueryExecutionContextBase" etc...

What's the name significance for consumers? Any public guidance on how applications can consume it?